mycore-solr/src/main/resources/components/solr/config/mycore.properties

```diff
@@ -149,3 +149,7 @@ MCR.Solr.AddDerivates.Fields=id,returnId,derivateMaindoc,iviewFile,derivateType
 #MCR.Object.IDMapper.Class=org.mycore.solr.idmapper.MCRSolrIDMapper
 #MCR.Object.IDMapper.ObjectSolrFields=doi,category,derivates,id
 #MCR.Object.IDMapper.DerivateSolrFields=category,derivateOrder
+
+# Workaround for https://issues.apache.org/jira/browse/SOLR-17502 .
+# Enabled by default, because stability is more important than performance.
+MCR.Solr.UseHttp1.1=true
```

Comment:
As we use "." for hierarchy I would prefer this Property to be named "MCR.Solr.UseHttp_1_1" instead.